docs/src/howto/storage.md

```diff
@@ -0,0 +1,36 @@
+# Using Default Storage in Canonical Kubernetes
+
+Canonical Kubernetes offers a local storage option to quickly set up and run a cluster, especially for single-node support. This guide walks you through enabling and configuring this feature.
```

Comment:
can we word wrap at 79 chars? It makes the diffs more manageable and doesn't trip up merging so often